README.rst

```diff
@@ -33,3 +33,14 @@ Bug tracker
 ===========
 A bug tracker is available at
 https://github.com/dataflake/dataflake.fakeldap/issues
+
+Building and Deploying
+======================
+
+Requirements:
+  - A Packagecloud.io key with Environment variable set to it, such as package_cloud_key="packagecloudkey"
```

Comment:
err, i meant Packagecloud.io and Environment, the variable does need to be all caps

Reply:
also `an environment`